app/controllers/AnnotationController.scala

```diff
@@ -105,24 +105,25 @@ class AnnotationController @Inject()(
     }
   }
 
-  @ApiOperation(hidden = true, value = "")
-  def infoWithoutType(annotationId: String, timestamp: Long): Action[AnyContent] =
-    sil.UserAwareAction.async { implicit request =>
+  @ApiOperation(value = "Information about an annotation", nickname = "annotationInfo")
+  @ApiResponses(
+    Array(new ApiResponse(code = 200, message = "JSON object containing information about this annotation."),
+          new ApiResponse(code = 400, message = badRequestLabel)))
+  def infoWithoutType(
+      @ApiParam(
+        value =
+          "For Task and Explorational annotations, id is an annotation id. For CompoundTask, id is a task id. For CompoundProject, id is a project id. For CompoundTaskType, id is a task type id")
+      id: String,
+      @ApiParam(value = "Timestamp in milliseconds (time at which the request is sent)", required = true) timestamp: Long)
+    : Action[AnyContent] = sil.UserAwareAction.async { implicit request =>
+    log() {
       for {
-        annotationIdValidated <- ObjectId.parse(annotationId)
-        annotation <- annotationDAO.findOne(annotationIdValidated) ?~> "annotation.notFound"
-        typ = if (annotation._task.isEmpty) AnnotationType.Explorational else AnnotationType.Task
-        restrictions <- provider.restrictionsFor(typ.toString, annotationId) ?~> "restrictions.notFound" ~> NOT_FOUND
```

Comment:
From what I can tell the main `info(...)` method, does the same checks for `restrictions`, `timespan etc. Not sure if this really needs to be duplicated in the `infoWithoutType` method. But maybe I am overlooking something.

Reply:
you’re right, forwarding this to the original info method should do the trick :+1: